src/server/config/env/runTime.js

```diff
@@ -236,6 +236,41 @@ const runTime = [
     defaultValue: 'false',
     normalize: (input) => (input.toLowerCase() === 'true' ? 'true' : 'false'),
   },
+  // OpenTelemetry Configuration
+  {
+    name: 'OTEL_EXPORTER_OTLP_LOGS_ENDPOINT',
+    validate: (input) => {
+      if (!input) return;
+      // eslint-disable-next-line no-new -- intentionally using new for side effect of validation
+      new URL(input);
```

Comment:
im assuming this throws an error for invalid URLs? Even though its more verbose, i'd prefer something like

```suggestion
try {
      new URL(input);
} catch (e) {
      throw new Error('OTEL_EXPORTER_OTLP_LOGS_ENDPOINT was set with an invalid URL');
}
```

It makes it much easier to read / understand

Reply:
This is how the existing error is output `'ERROR (OTEL_EXPORTER_OTLP_LOGS_ENDPOINT): Invalid URL'`

Reply:
Right, I expected the error message to be fine, but the readability of the code is what i'm questioning. All other validations in this file throw an error. This just has a floating initialization, and someone may not have the prior knowledge that `new URL` throws for invalid URLs.  Being explicit like the code above removes a lot of mental load. Or maybe even expanding the comment in the lint description to say 'this will throw if the URL is invalid'

Reply:
You don't think "intentionally using new for side effect of validation" is clear?

Reply:
I don't, but I also don't want to drag this on too long, the rest of the PR looks good to me. Im fine to leave it as-is